src/components/Courses.jsx

```diff
@@ -31,6 +55,13 @@ const Courses = () => {
                 <Accordion.Item>
                   <Accordion.Header>
                     {course.code}: {course.name}
+                    <Button variant="danger"
+                      onClick={
+                        (e) => handleClick(e, course.code)
```

Comment:
if you don't need the event logic, you can simiplify
`onClick={() => handleClick(course.code)}`